backend/core/socketio_config.py

```diff
@@ -21,10 +24,22 @@
     socketio_server=sio_server, socketio_path=SOCKETIO_PATH
 )
 
+cookie = SimpleCookie()
+
 
 @sio_server.on("connect")
 async def sio_connect(sid, environ):
-    logger.info(f"A user sid[{sid}] connected")
+    try:
+        cookie.load(environ["HTTP_COOKIE"])
```

Comment:
In this code, I try to ged the token from HTTP_COOKIE instead of send the token as a parameter, then decode the token to get the user phone number. After we get the user phone number, we can put that inside the socketio session to be use later in socketio event.

Reply:
Ah, what threw me was the name `environ`. I thought that meant `os.environ`, but in this case it is the HTTP request. Never mind, all good.